Microsoft.Toolkit.Uwp/Helpers/StreamHelper.cs

```diff
@@ -30,8 +31,9 @@ public static class StreamHelper
         /// Get the response stream returned by a HTTP get request.
         /// </summary>
         /// <param name="uri">Uri to request.</param>
+        /// <param name="cancellationToken">instance of <see cref="CancellationToken"/></param>
         /// <returns>Response stream</returns>
-        public static async Task<IRandomAccessStream> GetHttpStreamAsync(this Uri uri)
+        public static async Task<IRandomAccessStream> GetHttpStreamAsync(this Uri uri, CancellationToken cancellationToken)
```

Comment:
Only consumer of StreamHelper within SDK is ImageCache.. Agree there might be external users.. Can Ave will make it default / optional